apps/browser/components/Workspace.tsx

```diff
@@ -95,13 +101,11 @@ function CurrentWorkspace() {
                 </div>
               </>
             )}
-          </div>
-        </>
-      ) : (
-        <div className="m-2 mt-1 h-[20vh] w-[calc(100%-1rem)] animate-pulse rounded-lg bg-zinc-700" />
```

Comment:
we should use the loader that Colin added rather than the spinner

Reply:
Agreed, feel free to update